pkg/controller/manager/sync_action.go

```diff
@@ -138,7 +139,12 @@ func syncBackupTasks(clusterID string, cluster *v1.ScyllaCluster, syncer stateCa
 			update := false
 			for _, managerTask := range managerState.BackupTasks {
 				if managerTask.ID == backupTask.ID {
+					// Keep special "now" value evaluated on task creation.
+					if strings.HasPrefix(backupTask.SchedulerTaskSpec.StartDate, "now") {
```

Comment:
would it be better to make a function so you don't have to repeat the parsing and you can unit test it?

Reply:
done